docs/changelog.md

```diff
@@ -1,5 +1,9 @@
+
 <!-- vale Canonical.007-Headings-sentence-case = NO -->
 
+## [#178 chore: migrate architecture docs](https://github.com/canonical/github-runner-image-builder-operator/pull/178) (2025-12-17)
+- Moved charm-architecture.md from Explanation to Reference category.
```

Comment:
Is this really required? I thought we only add user-relevant and not chore's to the changelog